test/cql-pytest/test_secondary_index.py

```diff
@@ -345,6 +345,21 @@ def test_filter_cluster_key(cql, test_keyspace):
         rows = cql.execute(stmt)
         assert_rows(rows, [1, 1])
 
+# Reproduces #13548:
+# Create a huge index on a table and immediately recreate the table (drop and create)
+# With `--enable-keyspace-column-family-metrics 1` cmd line param this should slow down the view builder
+# (which is busy building the index), enough to get rid of the first table object from the `database` object
+# but also allow for creating another table with the same name, which raised a double-metrics-registration exception
+def test_instant_table_recreation(cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, 'p int primary key, v text') as table:
+        insert_statement = cql.prepare(f'INSERT INTO {table} (p,v) VALUES (?, ?)')
+        big_string = 'x'*665360
```

Comment:
Nitpick: you should be able to reduce this to 64K + 1 and it should fail in exactly the same way and be a bit faster.

Reply:
I decreased the value, but used `66536`, which is also used in 3 other tests reproducing the #8627 issue.